python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1236,44 +1236,44 @@ def run(self, *, ccdExposure, dnlLUT=None, bias=None, deferredChargeCalib=None,
             )
 
         if self.config.doAssembleCcd:
-            # Input units: ADU
+            # Input units: electrons
```

Comment:
With this here, this needs to state that we're in ADU still.

Reply:
thanks - updated